src/plugins/imagery/components/Compass/Compass.vue

```diff
@@ -62,6 +63,10 @@ export default {
     sizedImageDimensions: {
       type: Object,
       required: true
+    },
+    showCompassHUD: {
```

Comment:
Since `showCompassHUD` is bring provided by `ImageryView.js`, we could inject it here instead of using a prop.

Reply:
Wilco